recipes/r-comorbidity/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set version = '1.0.5' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-comorbidity
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/comorbidity_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/comorbidity/comorbidity_{{ version }}.tar.gz
+  sha256: 243d8c12901860ed5ccd5f7ec633467aff21d25164225ef8dab5ff9f06b41dd6
```

Do you need both? In that case, use:

```suggestion
    - url: {{ cran_mirror }}/src/contrib/comorbidity_{{ version }}.tar.gz
      sha256: 243d8c12901860ed5ccd5f7ec633467aff21d25164225ef8dab5ff9f06b41dd6
    - url: {{ cran_mirror }}/src/contrib/Archive/comorbidity/comorbidity_{{ version }}.tar.gz
      sha256: REPLACE_ME
```

---

TIL about that feature. Thanks!